### DISCUSSION TITLE
Neden İbax-io olmalı

### DISCUSSION THREAD

**Discussion by Emrekontas** [IBAX-io/go-ibax#1903] [Ecolibs]
Bence bu net gelecek beklentileri olan iyi bir proje. Umarım bu proje beklediğimiz gibi devam eder çünkü iyi bir ekip tarafından desteklenmektedir. Ayrıca ilerleyen zamanlarda bildiğimiz Avax ( Avalanche) veya bir Sol coin ( Solona) gibi fiyatlanabilir. Bu yüzden değerlendirmek gerek ekip başarılı bir şekilde çalışıyor.

**Comment by yolandadadada** [comment]
Could you speak English?
